recipes/emg3d/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 0a027ccae7f13cbbde903aab9fbedb89da97c72af90ddb566ddec09c0a5f8311
 
 build:
-  skip: True    # [win]  # Remove when numba-feedstock/pull/37 is merged.
-  # noarch: python       # Enable when numba-feedstock/pull/37 is merged.
+  skip: True  # [win]  # Remove when numba-feedstock/pull/37 is merged.
```

Comment:
Remove this too.

Reply:
OK. But it'll fail for the Windows builds then, because of missing numba.

Reply:
Yup. For noarch this is fine. Noarch recipes only build on linux so only the linux tests need to pass.

Reply:
Great, thanks, didn't know that.

So as long as a windows user installs numba on its own, it should not be a problem even as long as the numba builds are not generated for windows on conda-forge?